deploy/app.yaml

```
@@ -34,6 +34,8 @@ handlers:
   static_dir: vi
 - url: /static
   static_dir: static
+- url: /scriptor
+  static_dir: scriptor
```

```suggestion
- url: /scriptor/(.+)$
  static_files: scriptor/\1
  upload: scriptor/.+$
```
---


and can you please add in the Index-module: 
```py
    @exposed
    def scriptor(self):
        raise errors.Redirect("/scriptor/index.html")
```

This adds the possibility to open `/scriptor` instead of  `/scriptor/index.html`.